roles/metrics_server/tasks/main.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2025 VEXXHOST, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+- name: Ensure namespace exists (only if not kube-system)
+  run_once: true
+  when: metrics_server_helm_release_namespace != "kube-system"
+  kubernetes.core.k8s:
+    server_side_apply:
+      field_manager: "atmosphere.common"
+      force_conflicts: true
+    definition:
+      apiVersion: v1
+      kind: Namespace
+      metadata:
+        name: "{{ metrics_server_helm_release_namespace }}"
```

Comment:
I just thought about this, we can leverage flux's option for `createNamespace` and avoid this.  I created https://github.com/vexxhost/atmosphere.common/issues/33 for all the other ones.